apps/gro/controllers/address-start.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const BaseController = require('hof').controllers.base;
+const Client = require('node-rest-client').Client;
+const config = require('../../../config');
+
+module.exports = class AddressStartController extends BaseController {
+  constructor(options) {
+    super(options);
+  }
+
+  supercall() {
+    super.successHandler.apply(this, arguments);
+  }
+
+  getValues(req, res, callback) {
+    const postcode = req.sessionModel.attributes.postcode.toUpperCase();
+    req.sessionModel.set('postcode', postcode);
+    const previouspostcode = req.sessionModel.attributes.previouspostcode;
+    req.sessionModel.set('previous-step', res.locals.backLink);
+    if (postcode !== previouspostcode) {
+      const client = new Client();
+      const url = config.postcode.hostname + config.postcode.addresses.path;
+      const args = {
+        parameters: {},
+        headers: {'Authorization': config.postcode.authorization || ''}
```

Comment:
Does the postcode look up work without an authorisation token? Is there any point in giving the user the option to use an empty string as the token?